other/upgrade_2-1_postgresql.sql

```diff
@@ -306,6 +306,43 @@ if (@$modSettings['smfVersion'] < '2.1')
 ---}
 ---#
 
+---# Parsing board descriptions and names
+---{
+if (@$modSettings['smfVersion'] < '2.1')
```

Comment:
Same like mysql, not safe for 2.1 to 2.1 upgrade

Reply:
Keep in mind that we do not want to do this change during a 2.1 to 2.1 upgrade, this change should only be done from 2.0 and only once.